### ISSUE TITLE
Modify "Moving of the table" rules

### ISSUE THREAD

**Issue by Anirudh94** [Ian-Stewart-Binks/Foosball#21]
I think the rules should perhaps be modified so that it is something along the lines of:

Moving of the table:
- After a match has begun the table is not allowed to be moved
- Before a match has begun the table may be moved if all players agree to do so

**Comment by Ian-Stewart-Binks** [comment]
Agreed. But also - can the table be moved mid game if players agree on it in between points?

**Comment by Anirudh94** [comment]
Ok so we could change it to: "When the ball is not in play, if all players agree to do so, the table can be moved"? (or something to that effect)

**Comment by Ian-Stewart-Binks** [comment]
Something like that. I agree.
